jdaviz/configs/default/plugins/collapse/collapse.vue

```diff
@@ -49,5 +49,33 @@
       @click:action="collapse"
     ></plugin-add-results>
 
+    <j-plugin-section-header v-if="export_enabled">Results</j-plugin-section-header>
```

Comment:
```suggestion
    <j-plugin-section-header v-if="collapsed_spec_available && export_enabled">Results</j-plugin-section-header>
```

Reply:
i kept it like this to be consistant with the moment map plugin, which displays the 'results' section header if only export_enabled is true. should i change there too?

Reply:
I think so - it seems weird to just have the section header with nothing inside it 🤷‍♂️